### PR TITLE
Tim Scribble

### PR DESCRIPTION
**Pull request by tomiteo**
I know it's not great, but I've been under the weather and just had to crash last night when I got home.

Feeling very frustrated by rails. Feel like I'm just copying code from one place to another without understanding of the underlying principles. 
Feel like writing in rails is not making me a better programmer but just someone good at copying stuff and coloring within the lines. 

2/5s

### CODE REVIEW

**Comment by jshawl** [comment]
:+1: @timothytgregg if it makes you feel better, I get that feeling w/ rails too. Let's find a time to pair this week and talk about this!

File: scribbl/app/assets/stylesheets/posts.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the posts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (jshawl) [inline scribbl/app/assets/stylesheets/posts.scss:3]: avoid using generators when possible - this helps me avoid that "coloring within the lines" feeling

File: scribbl/app/controllers/application_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class ApplicationController < ActionController::Base
+  # Prevent CSRF attacks by raising an exception.
+  # For APIs, you may want to use :null_session instead.
+  protect_from_forgery with: :exception
```
Comment (jshawl) [inline scribbl/app/controllers/application_controller.rb:4]: you can use `authenticate_user` here to restrict creation/editing/deletion to logged in users

File: scribbl/app/views/posts/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<h1>Editing Post</h1>
+
+<%= form_for :post, url: post_path(@post), method: :patch do |f| %>
```
Comment (jshawl) [inline scribbl/app/views/posts/edit.html.erb:3]: recommend:

``` rb
form_for @post do |f| 
```

File: scribbl/app/views/posts/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<h1>Listing Posts</h1>
+<table>
```
Comment (jshawl) [inline scribbl/app/views/posts/index.html.erb:2]: avoid `rails g scaffold` here. Also tables should be reserved for tabular data

File: scribbl/db/seeds.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+])
+# 
+# comments = Comment.create([
+#   {text:"The first comment on the first post", post: posts[0]},
+#   {text:"The first comment on the second post", post: posts[1]},
+#   {text:"The first comment on the third post", post: posts[2]}
+# ])
```
Comment (jshawl) [inline scribbl/db/seeds.rb:14]: this looks good! why comment it out?

**Comment by jshawl** [comment]
:+1: You're off to a good start! Let me know if you'd like to work through user authentication and/or deployment together sometime this week

**Comment by tomiteo** [comment]
thanks Jesse! looks like we're meeting at 315 today. I'll try to do good work until then and come with good questions.

**Pull request closed without merging**